src/components/html/EcApplication.html

```diff
@@ -58,9 +73,20 @@
             </div>
           </article>
         </a-card>
+        <a-button
+          @click="$emit('click')"
```

Comment:
```suggestion
          @click="$emit('click:back')"
```

Só `click` é pouco descritivo demais, e se precisarmos adicionar outro evento `click` neste componente (click em outro elemento)?